Jenkinsfile.version

```diff
@@ -0,0 +1,59 @@
+def utils
+rstudioVersion = ''
+rstudioVersionMajor  = 0
+rstudioVersionMinor  = 0
+rstudioVersionPatch  = 0
+rstudioVersionSuffix = 0
```

Comment:
I think we should use environment variables rather than globals. Talking it over with Ben, he suggested that we should avoid global variables unless we need constants

Reply:
Sounds reasonable. Done!